src/app/systelab-components/select/gender-combobox.component.ts

```diff
@@ -55,4 +60,17 @@ export class GenderSelect extends AbstractComboBox<Element> implements AfterView
 	getIdField(): string {
 		return 'id';
 	}
+
+
+	deleteValueSelected(event: MouseEvent): void {
+		event.stopPropagation();
+		if (this.showAll) {
+			this._id = 'A';
+			this.description = this._descriptionAll;
+		} else {
+			this._id = 'U';
+			this.description = this._descriptionUnknown;
```

Comment:
To be coherent, and as the previous one:

why don't you do:
this._description = this._descriptionUnknown;
this.id = 'U';